### ISSUE TITLE
Add links to good resources about best practices for development in PHP

### ISSUE THREAD

**Issue by ttoine** [PrestaShop/docs#319]
In order to help people without a tech background to contribute with best practice code, let's add some links to interesting resources and example.

A few ideas:

- https://github.com/jupeter/clean-code-php
-https://refactoring.guru/ or https://sourcemaking.com/

Feel free to propose other links in a comment.

**Comment by justeen35** [comment]
> In order to help people without a tech background to contribute with best practice code

You want people not tech to contribute to code ?

That's a big challenge. Why bother ? Not tech people can contribute in many ways: write doc, tutorials, test software, discuss issues and features ...

**Comment by ttoine** [comment]
I agree. But sometimes, merchants also solve issues they meet while using the software and want to share what they did. They need some good advice to learn how to do it well. But also, they need resources to understand why it might not be possible to accept their contribution, because it does not meet the required quality.

Many merchants learned how to improve PrestaShop modules, themes, or core while using it, spending time on forums and user groups. They are not developers, that's right. But their knowledge is very valuable.

If it's not in the devdocs, they will have to find it by themselves. That's part of mentoring, and that will also help beginner devs, or seasoned devs that are starting to work with PrestaShop.
